opencga-app/app/scripts/docker/opencga/Dockerfile

```
@@ -13,8 +13,9 @@ ARG buildPath="./build"
 
 WORKDIR /opt/opencga
 ENV BASEDIR=/opt/opencga
+ENV OPENCGA_CONFIG_DIR=/opt/opencga/conf
 
-RUN adduser -u 1001 opencga
+RUN useradd -u 1001 opencga

```

Add -m to create home, verified:

```root@7bac65164994:/opt/opencga# ls /home
opencga```

```suggestion
RUN useradd -u 1001 -m opencga
```

---

Added thanks!